src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -28,7 +28,7 @@
  *
  * Parameter and service keys are case insensitive.
  *
- * A service id can contain lowercased letters, digits, underscores, and dots.
+ * A service id can contain letters, digits, underscores, and dots.
  * Underscores are used to separate words, and dots to group services
```

Comment:
this should also mention that this is only relevant when you have several services of the same class. otherwise you can use service id == class name.